### ISSUE TITLE
Updating to latest version

### ISSUE THREAD

**Issue by dyablohunter** [electroneum/electroneum#136]
Can you please provide detailed instructions of what a pool operator has to do to update the correct services and files to the latest version?

**Comment by Griunvaldas** [comment]
Just build from source or download builded binaries and replace your electroneumd with new version. Blockchain resync is not needed, I've had to bring my pool down for 2 minutes to update everything and now everything is running smoothly

**Comment by dyablohunter** [comment]
Thank you very much for reply, many pool operators will thank you for this. So only get the latest release of the "electroneumd" service?
P.S. I saw it has an "update" command, would it work or I need to replace it manually?

**Comment by Griunvaldas** [comment]
Electroneum-wallet-cli and other binaries you use won't make matters worse too ;)

**Comment by dyablohunter** [comment]
I am sorry but I don't understand your answer to my second post, there are 2 questions there. Thank you.

**Comment by Griunvaldas** [comment]
If you are building from source - git pull
If not - download latest binaries from releases and replace your files (don't forget to shut down wallet and daemon before replacing them)

**Comment by electroneumRepo** [comment]
@dyablohunter the "update" command will not install the update, You need to do it manually.
